docs/decisions/0013-custom-groups-for-enterprise-learners.rst

```diff
@@ -0,0 +1,98 @@
+0013 Creating Custom Groups for Enterprise Customers
+##############################
+
+Status
+******
+
+**Pending**
+
+Terminology
+*******
+*Enterprise Group* - Collection of enterprise customer users connected to a policy/subsidy
+
+*Membership* - Relationship between enterprise learner and enterprise group
+
+
+Context
+*******
+
+Groups is a new feature that will allow admins the flexibility to control and distribute funding. These changes will improve the user management and budgeting from the admin perspective, as well as adding personalization to the learner experience. Groups will be associated with a learner credit budget. In the past, we’ve used workarounds to sidestep this functionality, but with this new implementation, we will increase scalability and personalization.
+
+EnterpriseGroup
+*********************
+**Model properties**
+------
+- uuid, name, created, modified, history
+- enterprise_customer_uuid (FK to EnterpriseCustomer, related_name=”groups”)
+- policy_uuid
+
+**/api/v1/enterprise-group/<group_uuid>**
+The root URL for reading metadata about a group.
+
+**CRUD**
+------
+**api/v1/enterprise-group/?group_uuid=[group-uuid]&learner_uuid=[enterprise_customer_user_id]&enterprise_uuid=[enterprise_uuid]**
```

Comment:
group_uuid probably should be part of the path, not a query param

Reply:
No because in this case if they don't want to search by group uuid, they might want to search by enterprise uuid which would have multiple groups